pages/docs/concepts/producer.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Producer
+weight: 3
+---
+
+## What is a Producer? 
+
+An event indicates a state change or update which is triggered by a user's/device's action; this could be something like an item being placed in a shopping cart on an e-commerce website or clicking the subscribe button on a YouTube channel or a temperature change in the sensor. A producer is a type of application that senses that state change and then publishes that event as a message.
```

Comment:
```suggestion
An event indicates a state change or updates that is triggered by a user's/device's action; this could be something like an item being placed in a shopping cart on an e-commerce website, clicking the subscribe button on a YouTube channel, or a temperature change in a sensor. A producer is a type of application that senses that state change and then publishes that event as a message.
```